packages/driver/src/cy/commands/navigation.ts

```diff
@@ -15,10 +13,10 @@ import debugFn from 'debug'
 const debug = debugFn('cypress:driver:navigation')
 
 let id = null
-let previousDomainVisited = null
-let hasVisitedAboutBlank = null
-let currentlyVisitingAboutBlank = null
-let knownCommandCausedInstability = null
+let previousDomainVisited: boolean | null = null
+let hasVisitedAboutBlank: boolean | null = null
+let currentlyVisitingAboutBlank: boolean | null = null
+let knownCommandCausedInstability: boolean | null = null
```

Comment:
Seems like the defaults should be updated to false

Reply:
Done.